internal/parse/parser.go

```diff
@@ -277,6 +303,137 @@ func (p *Parser) parseGoFullName() (FullName, bool, error) {
 	return FullName{}, false, nil
 }
 
+// parseList takes a parsing function that returns a FullName and parses a
+// bracketed, comma seperated, list. On success it returns an array of FullName
+// objects. Otherwise it returns an empty array, false and an error.
+func (p *Parser) parseList(parseFn func(p *Parser) (FullName, bool, error)) ([]FullName, bool, error) {
+	cp := p.save()
+	if p.skipByte('(') {
+		parenPos := p.pos
+		p.skipSpaces()
+
+		nextItem := true
+		var objs []FullName
+		for nextItem {
+			if obj, ok, err := parseFn(p); ok {
+				objs = append(objs, obj)
+			} else if err != nil {
+				return nil, false, err
+			} else {
+				return nil, false, fmt.Errorf("invalid identifier near char %d", p.pos)
+			}
+			p.skipSpaces()
+			if p.skipByte(')') {
+				return objs, true, nil
+			}
+			nextItem = p.skipByte(',')
+			p.skipSpaces()
+		}
+		return nil, false, fmt.Errorf("missing closing parentheses for char %d", parenPos)
```

Comment:
I think either "at char" is slightly clearer, or perhaps "near char" is more consistent with the other error messages here